app/models/manageiq/providers/redhat/infra_manager/provision/state_machine.rb

```diff
@@ -40,10 +40,14 @@ def customize_destination
       _log.info("#{message} #{for_destination}")
       update_and_notify_parent(:message => message)
       configure_container
-      signal :poll_destination_powered_off_in_provider
+      signal :customize_guest
     end
   end
 
+  def customize_guest
+    signal :poll_destination_powered_off_in_provide
```

Comment:
Ah, my fault ：）. By the way, I have not used the travis-ci before, so it is a challenge for me to write proper Rspec test. As the result the TC always fails due to some test related code, is this necessary?